app/assets/javascripts/discourse/app/templates/group-activity-posts.hbs

```diff
@@ -1,14 +1,5 @@
-{{#if this.canLoadMore}}
-  {{hide-application-footer}}
-{{/if}}
-
-<LoadMore @selector=".user-stream-item" @action={{action "loadMore"}}>
-  <div class="user-stream">
-    {{#each this.model as |post|}}
-      <GroupPost @post={{post}} />
-    {{else}}
-      <div>{{i18n this.emptyText}}</div>
-    {{/each}}
-  </div>
-  <ConditionalLoadingSpinner @condition={{this.loading}} />
-</LoadMore>
\ No newline at end of file
+<PostList
+  @posts={{this.model}}
+  @fetchMorePosts={{this.fetchMorePosts}}
+  @emptyText={{i18n "groups.empty.posts"}}
+/>
```

Comment:
What is your plan here regarding the class names, not necessarily in this PR, but next up. 

I see we are moving from `user-stream-item` to `post-list-item` for this group view, but do you think we should replace `user-stream-item` in the user activity feed as well? Curious how many themes/components style those selectors.